node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -89,6 +98,51 @@ func NewOsDeviceConnectivityHelperScsiGeneric(executer executer.ExecuterInterfac
 	}
 }
 
+func (r OsDeviceConnectivityHelperScsiGeneric) GetMpathdOutputByVolumeIds(volumeIdVariations []string) (string, error) {
+	return r.Helper.GetMpathdOutputByVolumeId(volumeIdVariations)
+}
+
+func (r OsDeviceConnectivityHelperScsiGeneric) GetMpathDeviceName(volumePath string) (string, error) {
```

Comment:
I think we have a [naming issue](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/dm_multipath/multipathd_commands) here:
```
# multipathd show wildcards
multipath format wildcards:
%n  name
%w  uuid
%d  sysfs
```
existing [`mpathDeviceName`](https://github.com/IBM/ibm-block-csi-driver/blob/v1.9.0/node/pkg/driver/node_test.go#L79)/[`mpathDevice`](https://github.com/IBM/ibm-block-csi-driver/blob/v1.9.0/node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go#L232)/[`baseDevice`](https://github.com/IBM/ibm-block-csi-driver/blob/v1.9.0/node/pkg/driver/node.go#L169) (e.g. `dm-0`) -> `dmSysFsName`?
new [`mpathDeviceName`](https://github.com/IBM/ibm-block-csi-driver/blob/35e2502d432c6e502d4d1175b2459dd5c41a25d9/node/pkg/driver/node_utils.go#L602) (e.g. `mpathc`) -> `dmName`?
we can align the existing in a separate ticket, but I don't think we should use the same name for this new entity

Reply:
so maybe just rename the existing [`mpathDeviceName`](https://github.com/IBM/ibm-block-csi-driver/blob/v1.9.0/node/pkg/driver/node_test.go#L79) (which is a small diff), so the word "name" would (temporarily) be used only for this new entity (e.g. `mpathc`), and open a ticket to align the naming of the other existing places (for e.g. `dm-0`)?